subsys/bluetooth/host/gatt.c

```diff
@@ -3682,7 +3863,21 @@ int bt_gatt_clear(u8_t id, const bt_addr_le_t *addr)
 		return err;
 	}
 
-	return bt_gatt_clear_cf(id, addr);
+#if defined(CONFIG_BT_GATT_SERVICE_CHANGED)
+	err = bt_gatt_clear_sc(id, addr);
+	if (err < 0) {
+		return err;
+	}
+#endif /* CONFIG_BT_GATT_SERVICE_CHANGED */
+
+#if defined(CONFIG_BT_GATT_CACHING)
```

Comment:
Im seeing this right or there 2 times the same code in here?

Reply:
Done in last push!